### DISCUSSION TITLE
杭州市曲江新鸥鹏第三城停贷风险告知书

### DISCUSSION THREAD

**Discussion by jadesnake** [WeNeedHome/SummaryOfLoanSuspension#1094] [General]
![79d41cd1548f2f607fc3e8f508cc8e6](https://user-images.githubusercontent.com/16377295/183054494-10bdd2dd-1b65-4558-96c7-a544e93d357c.jpg)

**Comment by yuhan6665** [comment]
帮你提交到首页了

**Reply by Tianlong-995** [reply]
干脆开盘第一天就发停贷通知书算了

**Reply by yuhan6665** [reply]
难道正解不是交屋时才办贷款吗。。一手交钱一手交货
